src/api/stats/getAmmPrices.ts

```diff
@@ -580,7 +585,35 @@ const updateAmmPrices = async () => {
       };
     };
 
-    const ammPrices = fetchAmmPrices(pools, knownPrices);
+    // All LP oracleIds should be set to the native assets, the wrapped version of each of those is then set based on those to avoid different values
+    const ammPrices = fetchAmmPrices(pools, knownPrices).then(prices => {
+      const gasTokens = [
```

Comment:
Suggestion to import data from address book, no need to list native coins manually

Reply:
There's no reference in the addressbook to a token being native

Reply:
we usually also have the native key referencing the wrapped token

`const BNB = {
  name: 'WBNB Token',
  symbol: 'WBNB',
  address: '0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c',
  chainId: 56,
  decimals: 18,
  website: 'https://www.binance.com/',
  description:
    'Binance Coin (BNB) is an exchange-based token created and issued by the cryptocurrency exchange Binance. Initially created on the Ethereum blockchain as an ERC-20 token in July 2017, BNB was migrated over to Binance Chain in February 2019 and became the native coin of the Binance Chain.',
  logoURI:
    'https://tokens.pancakeswap.finance/images/0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c.png',
} as const;`

`BNB,
  WBNB: BNB,`

Reply:
Not entirely true. It's possible to get those via `WNATIVE`. Not perfect but better than having a hardcode here.

It's so much work to bring new networks, would be great to have one thing less to keep in mind.